test/setup.jest.ts

```diff
@@ -3,7 +3,14 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-// import '@testing-library/jest-dom/extend-expect';
+import '@testing-library/jest-dom/extend-expect';
+import { ReadableStream } from 'web-streams-polyfill'; // Polyfill for ReadableStream
+import { TextDecoder, TextEncoder } from 'util';
+global.TextEncoder = TextEncoder;
+// @ts-expect-error
+global.TextDecoder = TextDecoder;
+// @ts-expect-error
+global.ReadableStream = ReadableStream;
```

Comment:
I tested locally and we don't need these changes

Reply:
addressed